tests/test_board.py

```diff
@@ -1,60 +1,142 @@
 import unittest
+
 from rookpoly import Board
-from rookpoly import Polynomial
+from rookpoly.polynomial import Polynomial
 
 
 class TestBoard(unittest.TestCase):
-
     empty_boards_limit = 7
 
     def setUp(self):
         self.empty_boards = [Board(['0' * i for _ in range(i)]) for i in range(self.empty_boards_limit)]
-        self.board_1 = Board(["110", "100", "000", "001"])
-        self.board_2 = Board(["00111", "00111", "11100", "11100", "11000"])
+        self.board_global_1 = Board(["110", "100", "000", "001"])
+        self.board_global_2 = Board(["00111", "00111", "11100", "11100", "11000"])
+
+    def test_init(self):
+        # Testowanie poprawnej inicjalizacji planszy
```

Comment:
We should probably rewrite comments in English

Reply:
Reviewed prior to [b5cdb71](https://github.com/Herman0802/RookPoly/pull/3/commits/b5cdb7161e9e8f2428672f5e2ab6bdea3f9d9879)